services/venice-router/src/test/java/com/linkedin/venice/router/throttle/RouterRequestThrottlingTest.java

```diff
@@ -72,7 +74,12 @@ public static Object[][] requestType() {
     return new Object[][] { { RequestType.MULTI_GET }, { RequestType.COMPUTE } };
   }
 
-  @Test(timeOut = 30000, dataProvider = "multiGet_compute")
+  /** A simple Runnable-like interface just to reduce boilerplate in the test... */
+  private interface ScatterCall {
+    void run() throws RouterException;
+  }
```

Comment:
I should be doing this more often...

Reply:
Yeah... test code is very repetitive... it's nice to DRY it up a bit :D ...